autodistill/core/custom_detection_model.py

```diff
@@ -1,15 +1,20 @@
+import numpy as np
 import supervision as sv
 from PIL import Image
 
+from autodistill.detection.detection_base_model import DetectionBaseModel
 
-class CustomDetectionModel:
+
+class CustomDetectionModel(DetectionBaseModel):
     """
     Run inference with a detection model then run inference with a classification model on the detected regions.
     """
 
-    def __init__(self, detection_model, classification_model):
+    def __init__(self, detection_model, classification_model, set_of_mark=None):
```

Comment:
I think this should be `set_of_marks` plural vs singular (you add a "set of marks" to do "set of mark prompting")